Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue7890.cs

```diff
@@ -46,6 +59,22 @@ protected override void Init()
 			};
 		}
 
+#if UITEST
+		[Test]
+		public void RefreshingListViewCrashesWhenDisposedTest()
+		{
+			foreach (var c in Enumerable.Range(0, Count))
+			{
+				RunningApp.WaitForElement(q => q.Marked(c.ToString()));
+			}
+			RunningApp.Tap(q => q.Button("RemoveBtn"));
+			foreach (var c in Enumerable.Range(RemoveFrom, RemoveCount))
+			{
+				RunningApp.WaitForNoElement(q => q.Marked(c.ToString()));
+			}
```

Comment:
What do you think this is doing?

I think you should just do a `WaitForElement` on the remove button, then tap that and then verify that the range that you expect to be deleted is actually gone through an Assert statement. I don't think like this the test has the desired effect.

Are you able to run the test locally?

Reply:
> I think you copied a bit too much from where you got this ;)

Looks like that )

> Are you able to run the test locally?

After some research and fighting with adb, yes )

I've fixed UI test